spec/defines/dns__server__options_spec.rb

```diff
@@ -111,6 +111,16 @@
     it { should contain_file('/etc/bind/named.conf.options').with_content(/allow-query/)  }
 
   end
+  context 'disable dns-sec option' do
```

Comment:
Thank you for writing this!

For style can you add a newline between the contexts and remove the neline on line 122?
(to give each context a more "self-contained" feeling)